kubernetes/adapter.go

```diff
@@ -26,27 +26,14 @@ type Adapter struct {
 	fabricSupport                  bool
 }
 
-type ingressType int
+type IngressType string
 
 const (
-	ingressTypeIngress ingressType = iota + 1
-	ingressTypeRouteGroup
-	ingressTypeFabric
+	TypeIngress    IngressType = "ingress"
+	TypeRouteGroup IngressType = "routegroup"
+	TypeFabric     IngressType = "fabric"
```

Comment:
maybe should name this `TypeFabricGateway IngressType = "fabricgateway"` for consistency

Reply:
yes I think TypeFabricGateway makes sense

Reply:
Renamed